docs/api/apiv3/components/examples/hierarchy_item_collection_filtered_response.yml

```diff
@@ -0,0 +1,53 @@
+description: |-
+  Filtered response of a hierarchy structure starting at a specific parent and restricting the depth.
+value:
+  _type: Collection
+  total: 3
+  count: 3
+  _embedded:
+    elements:
+      - _type: HierarchyItem
+        id: 1338
+        label: Stormtroopers
+        short: ST
+        _links:
+          self:
+            href: /api/v3/custom_field/42/items/1338
+          parent:
+            href: /api/v3/custom_field/42/items/1337
+          children:
+            - href: /api/v3/custom_field/42/items/1340
+              name: ST-377
+            - href: /api/v3/custom_field/42/items/1341
+              name: ST-422
```

Comment:
I think you're right. Looks simple and elegant.

![image](https://github.com/user-attachments/assets/7bca08ab-310f-4b38-a91e-b2ba9d1f184d)